### PR TITLE
Document project members

### PR DESCRIPTION
**Pull request by dholbach**
As a follow-up to #240, document current Flux Project Members to make it easier for new applicants to find out who could be a sponsor.

~~Only merge after #240 has been landed.~~

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Review by kingdonb — APPROVED** [review]
👍 

> who are not maintainers (yet)
💯

**Comment by kingdonb** [comment]
I think my review did not count because I haven't been granted Write access here yet after this one merged:

* https://github.com/fluxcd/community/pull/233

@fluxcd/core-maintainers Could someone please take a look and update the repo permissions for `fluxcd/community`?

**Comment by stefanprodan** [comment]
> https://github.com/orgs/fluxcd/teams/core-maintainers Could someone please take a look and update the repo permissions for fluxcd/community?

Done

**Pull request merged**
